docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+---
+sidebar_position: 4   
+description: Changing the suit's name and description 2 
+---
+
+import './customhover.css';
+import Tooltip from './tooltip';
+
+
+# Adding Localisation Overrides
+
+*In this tutorial, you’ll learn how to modify suit names and descriptions in the game using* 
+> <Tooltip text={<><strong>Localisation Overrides</strong><span className="icon-alert"></span></>} tooltip="A way to replace or modify in-game text, like names or descriptions, with your own custom text." />. 
+
+You'll learn how to:
+- locate the necessary files, 
+- identify and convert *"strings"*, 
+- update the manifest with your custom text. 
+
+>By the end, you'll have a fully customized suit name and description in the game.
+
+*When necessary, there will be a link to the ***advanced search section*** to change any other text as well*
+
+
+<details>
+<summary>**About Tooltips**</summary>
+:::info tooltips
+*To assist you*, I’ve added <Tooltip
+  text={<strong>tooltips</strong>}
+  tooltip="This is a tooltip"
+/>
+ throughout the tutorial to explain any tricky terms as you encounter them. 
+
+<Tooltip text={<><strong>Key terms</strong><span className="icon-alert"></span></>} tooltip="These will need the most attention" /> are marked with an exclamation icon when they're necessary. <Tooltip text={<italics>Hover over almost any</italics>} tooltip="almost" /> **bold** term for a quick explanation. Highlighted terms become easier to understand over the course of the tutorial.
+
+    >Explore the tooltips as needed for clarification
+:::
+</details>
+
+You will need: 
+-  <Tooltip text={<strong>GlacierKit</strong>} tooltip="A modding tool used for editing and managing game files in Hitman: World of Assassination." />
```

Comment:
This terminology should have been introduced long before the user gets to this tutorial, so I can't see a tooltip being super useful here